src/Components/CloudAccounts/NoCloudAccounts.tsx

```diff
@@ -0,0 +1,11 @@
+import { Content, EmptyState, EmptyStateBody } from '@patternfly/react-core';
+import { WrenchIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+export const NoCloudAccounts = () => (
+  <EmptyState icon={WrenchIcon} titleText="No cloud accounts">
+    <EmptyStateBody>
+      <Content>You do not have access to any cloud accounts</Content>
```

Comment:
Let's get clarification on the wording for this empty state - I'm not sure if "access" is the right word to use.

It may end up being "You do not have any cloud accounts connected, add there <here>" with a link